.github/workflows/causal_lm_cpp.yml

```diff
@@ -799,17 +799,27 @@ jobs:
           && ./build/samples/cpp/visual_language_chat/visual_language_chat ./llava_1_5_7b_ov/ monalisa.jpg
           <<< $'Who drew this painting?\nWhen did the painter live?'
         timeout-minutes: 4
-      - name: Download and convert LLaVa-Next model and an image
+      - name: Download and convert LLaVa-Next model
         run: |
           source ./ov/setupvars.sh
           optimum-cli export openvino --model llava-hf/llava-v1.6-mistral-7b-hf ./llava_v1_6_mistral_7b_ov/
-          wget https://llava-vl.github.io/static/images/monalisa.jpg
       - name: Run visual_language_chat C++ sample - LLaVa-Next
         run: >
           source ./ov/setupvars.sh
           && ./build/samples/cpp/visual_language_chat/visual_language_chat ./llava_v1_6_mistral_7b_ov/ monalisa.jpg
           <<< $'Who drew this painting?\nWhen did the painter live?'
         timeout-minutes: 4
+      - name: Download and convert InternVL2 model
+        run: |
+          source ./ov/setupvars.sh
+          python -m pip install timm
```

Comment:
Probably, `timm` package is missing due to installing `"optimum<1.23"` with `--no-dependencies` flag above. AFAICS it is required for exporting InternVL2 model AFAICS.
Should be moved it to "Install dependencies" step above?

Reply:
1. `optimum` is installed with `python -m pip install --upgrade-strategy eager -r ./samples/requirements.txt opencv-python --pre --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly` so it should bring `timm`.
2. Later `transformers` from branch is installed.
3. There's compatibility issue and `optimum` is downgraded to solve this, but other packages aren't affected thanks to `--no-dependencies`.

If `timm` isn't installed, it should be added to samples and tests `requirements.txt`.

Reply:
Moved to samples requirements